packages/react-dom-bindings/src/client/ReactDOMComponent.js

```diff
@@ -377,6 +475,50 @@ function setProp(
       domElement.setAttribute(key, sanitizedValue);
       break;
     }
+    case 'action':
+    case 'formAction': {
+      // TODO: Consider moving these special cases to the form, input and button tags.
```

Comment:
I decided to leave these here for now and not special case per tag. To keep the code smaller. Ideally these should follow the pattern of `input`, `select` and `textarea` instead.

That way we could more easily clear out props like `target`, `method` and `encType` when they're not relevant or set them to what they're supposed to be. That's what Fizz does. That probably also lead to hydration warnings but they also warn just in general.